.github/workflows/style.yml

```diff
@@ -28,3 +28,4 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install && npm run lint
+    - run: npx npcheck --github-token ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The existing workflow is set up to run when a file under `docs` changes (either via a push or pull request to `main`). When/how often do we want to run these license checks?

Reply:
I think running when the docs change is good since that is where we list the modules. I think we probably want to configure to run weekly as well.

Reply:
I'd suggest breaking this out into a separate workflow then, unless we think we'd want to run the existing lint checks weekly as well?

Reply:
Ideally every time we add a new module ?? 🤷‍♂️

Reply:
@richardlau how we can configure the `style.yaml` file in order to run it once a week and every time we update the docs ??

Reply:
Add an `on` `schedule` trigger (keeping the existing `push` and `pull_request` ones): https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule
e.g. https://github.com/nodejs/node/blob/44ffddae8b73a35704dfd4124ab4ce738cf546df/.github/workflows/close-stalled.yml#L3-L4
You can use https://crontab.guru/ to decipher cron syntax. e.g. `0 0 * * 1` would run every Monday at midnight.

We could also add a `workflow_dispatch` trigger if we ever want to manually run the workflow (without having to push/open a pull request).